addons/website_event/views/event_templates_page.xml

```diff
@@ -68,8 +91,40 @@
                 </div>
             </div>
         </nav>
-        
+        <!-- ======== -->
     </div>
+
+    <!-- Desktop -->
+    <section id="o_wevent_submenu_desktop" class="d-none d-lg-block mt-3 mb-3 mb-lg-2">
+        <div class="container">
+            <div class="d-flex align-items-baseline justify-content-between">
+                <nav class="d-flex flex-wrap justify-content-between align-items-center gap-2 flex-grow-1">
+                    <ul class="breadcrumb p-0 m-0">
+                        <li class="breadcrumb-item"><a href="/event" title="Back to all events">Events</a></li>
+                        <li class="breadcrumb-item active" aria-current="page"><span class="pe-3" t-field="event.name"/></li>
+                    </ul>
+
+                    <ul class="nav" t-att-data-menu_name="editable and 'Event Menu'" t-att-data-content_menu_id="editable and event.menu_id.id">
+                        <t t-foreach="event.menu_id.child_id" t-as="submenu">
+                            <t t-call="website.submenu">
+                                <t t-set="item_class" t-value="'nav-item'"/>
+                                <t t-set="link_class" t-value="'nav-link'"/>
+                            </t>
+                        </t>
+                    </ul>
+                </nav>
+                <!-- Workaround to set the navbar's height equal to the register page CTA -->
+                <span t-if="not event.menu_register_cta or event.is_participating or hide_register_cta" aria-hidden="true" class="btn invisible w-0 px-0">&#160;</span>
+
+                <a t-else=""
+                    t-att-href="'/event/%s/register' % (slug(event))"
+                    t-attf-class="btn btn-primary ms-3"
+                    title="Registration">
+                    Registration
+                </a>
```

Comment:
Now that the CTA is sometimes right next to the submenu, maybe it would make sense to hide the submenu when the CTA is displayed? (`t-if="not show_register_cta or submenu.name != 'register'"`)

Reply:
Yes, I've tried multiple approaches but without success 🤔
`submenu.name` returns  a `' Register '` string, with empty character (also, I assume that's "translatable")... do you have any hint?

Reply:
@stefanorigano 
Oh right my bad, it's actually `menu_type != 'register'` not `name`